11ty/definitions/split-attraction-model.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Split Attraction Model (SAM)
+slug: split-attraction-model
+defined: true
+excerpt: a way of describing attraction as two separate things—romantic attraction and sexual attraction. Used by many (but not all) ace-spec and aro-spec people, the split attraction model is one way to describe not experiencing both types of attraction, or not experiencing both in the same way—for example, an asexual person who experiences romantic attraction or an aromantic person who experiences sexual attraction.
+speech: noun
+reading:
+  - text: LGBTA Wiki
+    href: https://lgbta.wikia.org/wiki/Split_Attraction_Model_(SAM)
+---
+
+a way of describing attraction as two separate things: romantic attraction and sexual attraction. Used by many (but not all) ace-spec and aro-spec people, the split attraction model is one way to describe not experiencing both types of attraction, or not experiencing both in the same way—for example, an asexual person who experiences romantic attraction or an aromantic person who experiences sexual attraction.
```

Comment:
1. Could you please break the "Used by many..." into a separate `## Note`section and elaborate a bit more? I think we could also note that many bi and pan people also identify using this model too.
2. I think it would be great to spell out `ace-spec` first as `asexual-spectrum (ace-spec)` and`aro-spec`as `aromantic-spectrum` to help clarify for folks from outside the community.